Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/Rumble.java

```diff
@@ -17,11 +17,14 @@
 
 public class Rumble
 {
+  private static long timestamp;
```

Comment:
to keep it in line with the variable naming convention in the rest of the file, `timeStamp`